File: backend/go/models/tournament.go
package models

type Tournament struct {
	ID        string   `json:"id"`
	Name      string   `json:"name"`
	Game      GameType `json:"game"`
	Status    string   `json:"status"` // "running", "upcoming", "past"
	BeginAt   string   `json:"beginAt"`
	EndAt     string   `json:"endAt"`
	League    string   `json:"league"`
	ImageURL  string   `json:"imageUrl"`
}

File: frontend/react/src/types/tournament.ts
import type { GameType } from './match';

export interface Tournament {
  id: string;
  name: string;
  game: GameType;
  status: 'running' | 'upcoming' | 'past';
  beginAt: string;
  endAt: string;
  league: string;
  imageUrl: string;
}
